Tasks/DotnetCLIRestore/task.json

```diff
@@ -0,0 +1,159 @@
+{
+    "id": "9fac244b-8d7c-4d8e-a003-2097daa3270f",
+    "name": "DownloadGitHubNugetPackage",
+    "friendlyName": "Download Nuget GitHub Packages",
+    "description": "Restore your nuget packages using dotnet CLI",
+    "author": "Microsoft Corporation",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/build/dotnet-core-cli",
+    "helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?linkid=832194) or [see the .NET Core documentation](https://docs.microsoft.com/dotnet/core/)",
+    "category": "Build",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "runsOn": [
+        "Agent"
+    ],
+    "demands": [],
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "minimumAgentVersion": "2.115.0",
+    "instanceNameFormat": "dotnet $(command)",
+    "groups": [
+        {
+            "name": "restoreAuth",
+            "displayName": "Feeds and authentication",
+            "isExpanded": true,
+            "visibleRule": "command = restore"
+        },
+        {
+            "name": "restoreAdvanced",
+            "displayName": "Advanced",
+            "isExpanded": false,
+            "visibleRule": "command = restore"
+        }
+    ],
+    "inputs": [
+        {
+            "name": "packageName",
+            "type": "string",
+            "label": "Package Name",
+            "required": true,
+            "helpMarkDown": "Name of the package to download from github"
+        },
+        {
+            "name": "version",
+            "type": "string",
+            "label": "Package Version",
+            "required": true,
+            "helpMarkDown": "Version of the package to download from github"
+        },
+        {
+            "name": "externalEndpoints",
+            "aliases": [
+                "externalFeedCredentials"
+            ],
+            "type": "connectedService:github",
+            "label": "Credentials for feeds outside this organization/collection",
```

Comment:
wrong label which talks about ADO org/collection. this is purely github endpoint